### DISCUSSION TITLE
download speed of master build slower than the 2.94 version.

### DISCUSSION THREAD

**Discussion by odkrys** [transmission/transmission#2791] [Q&A]
This is a question, not an issue or bug report.

I am using openwrt on j4105 box and running transmission version 2.94.
The transmission is single-threaded and reaches up to 42MB/s at 100% usage of one core. (500mbps internet, 3 torrents)

I've seen recently transsmission ported to C++ and got a lot of commits, so I compiled and ran the latest commit version today.
(44e30bf092effc1daf237b209fc80d80e8494c39)

The download speed is only 30MB/s when 1 cpu usage reaches 100%. (3 same torrents)

Does it happen to other people too?

**Comment by KiralyCraft** [comment]
I also see this behavior. I've recently migrated my seedbox and therefore my Transmission version; It looks like only one core is used, and it shows in my download speeds, unfortunately.
